packages/integration-test/src/setup.js

```diff
@@ -231,6 +263,16 @@ class Setup {
              * @param {BN} scalingFactor - factor to convert between AZTEC note value and ERC20 token value
              */
             scalingFactor: ERC20_SCALING_FACTOR,
+
+            /**
+             * @param {Object} proxyAddresses - addresses of the deployed account registry proxy contracts for each network. 
+             * Note, these proxy addresses are for the 1.17.0 '@aztec/contract-addresses' package 
+             */
+            proxyAddresses: {
```

Comment:
I know this is brittle, and would need manually updating if we were to re-deploy. The `proxyAddress`s should ideally also be recorded in the `contract-addresses` package and fetched from there when needed. 

However, given that the proxy's are deployed via the `AccountRegistryManager` (in it's constructor) we'd need a new, custom script to collect the proxy addresses - they don't have their own migration/deploy so aren't compatible with the current flow.

Reply:
Makes sense, and hopefully these things don't change.